docs/articles/commands/custom_context_checks.md

```diff
@@ -87,6 +141,12 @@ A single check class can also implement multiple checks, like so:
 public class Check : IContextCheck<FirstAttribute>, IContextCheck<SecondAttribute>;
 ```
 
+or even multiple different kinds of checks, like so:
+
+```cs
+public class Check : IContextCheck<FirstAttribute>, IParameterCheck<SecondAttribute>;
+```
+
 This means that all other code in that class can be shared between the two check methods, but this should be used with caution - since checks are registered per type, you lose granularity over which checks should be executed; and it means the same construction ceremony will run for both checks.
 
-There is no limit on how many different checks can reference the same attribute, they will all be supplied with that attribute. Checks targeting `UnconditionalCheckAttribute` will always be executed, regardless of whether the attribute is applied or not.
\ No newline at end of file
+There is no limit on how many different checks can reference the same attribute, they will all be supplied with that attribute. Checks targeting `UnconditionalCheckAttribute` will always be executed, regardless of whether the attribute is applied or not. This does not exist for parameter checks.
```

Comment:
```suggestion
There is no limit on how many different checks can reference the same attribute, they will all be supplied with that attribute. Checks targeting `UnconditionalCheckAttribute` will always be executed, regardless of whether the attribute is applied or not. Unconditional context checks are not available for parameter checks.
```

Though perhaps we should move unconditional checks to before parameter checks is explained?